docs/readthedocs/source/doc/Nano/QuickStart/pytorch_inference.md

```diff
@@ -1,8 +1,215 @@
 # BigDL-Nano PyTorch Inference Overview
 
-add a link for examples here.
+BigDL-Nano provides serveral APIs which can easily apply optimizations on inference pipeline to improve latency and througput. Currently, performance accelerations are achieved by integrating extra runtime as inference backend engine or using quantization methods on full-precision trained models to reduce computation during inference. Trainer (`bigdl.nano.pytorch.Trainer`) provides the APIs for all optimizations you need for inference.
 
-###  Runtime Acceleration
-onnx runtime, openvino
+For runtime acceleration, BigDL-Nano has enabled two kinds of runtime for users in `Trainer.trace()`, ONNXRuntime and OpenVINO.
 
-### Quantization
\ No newline at end of file
+For quantization, BigDL-Nano provides only post-training quantization `trainer.quantize()` for users to inference with models of 8-bit precision. Quantization-Aware Training is not available for now. Model conversion to 16-bit like BF16, FP16 will be coming soon.
```

Comment:
"provides only post-training quantization `trainer.quantize()`" -> "provides only post-training quantization in `trainer.quantize()`"

Are we going to support FP16?

Reply:
yes. we will support FP16.